src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
+  width: 200px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    color: #616070;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    color: $main-text-color;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-top: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+}
+
+.product-cardprice {
```

Comment:
There seems to be a typo in the class name. It should be '.product-card__price' instead of '.product-cardprice' to maintain consistency with BEM naming conventions.